erizo_controller/erizoJS/erizoJSController.js

```diff
@@ -141,6 +143,14 @@ exports.ErizoJSController = function () {
         callback('callback', {type: 'initializing'});
     };
 
+    closeWebRtcConnection = function (wrtc) {
+        var associatedMetadata = wrtc.metadata || {};
+        wrtc.close();
+        log.info('message: WebRtcConnection status update, ' +
```

Comment:
Just a doubt: should we wait to a callback form wrtc confirming that it has been successfully closed?